compiler/circle-execution-plan/src/ExecutionPlanner.cpp

```diff
@@ -18,6 +18,11 @@
 #include <loco/IR/Algorithm.h>
 #include <luci/UserSettings.h>
 
+#include <boost/property_tree/ptree.hpp>
```

Comment:
can you use another library than boost?
currently there exist boost in infra but only `nnc` module uses it.
I don't know our release package has checked with boost package and don't have any plans to maintain version upgrade for boost.
If `circle-execution-plan` is to be released in our package someday then please reconsider this or if you are sure this won't be released then this can go as is.

CC @binarman

Reply:
Please request another review from whom is able to. I didn't follow this module so please go on without my approval.

Reply:
Thank you for your comments! I will discuss it with @binarman and if it will be necessary I will rewrite this PR

Reply:
`circle-quantizer` uses `jsoncpp`.
I think this component could use it as well